src/resolve/resolved-timeouts.h

```diff
@@ -19,12 +19,35 @@
 /* Maximum attempts to send MDNS requests, see RFC 6762 Section 8.1 */
 #define MDNS_TRANSACTION_ATTEMPTS_MAX 3
 
-#define TRANSACTION_ATTEMPTS_MAX(p) (\
-                                     (p) == DNS_PROTOCOL_LLMNR ?        \
-                                     LLMNR_TRANSACTION_ATTEMPTS_MAX :   \
-                                     (p) == DNS_PROTOCOL_MDNS ?         \
-                                     MDNS_TRANSACTION_ATTEMPTS_MAX :    \
-                                     DNS_TRANSACTION_ATTEMPTS_MAX)
+/* Maximum attempts to send an mDNS continuous query.
+ *
+ * RFC 6762 Section 5.2 does not specify a maximum number of attempts directly.
+ * However, it outlines two important guidelines:
+ *
+ * 1. The interval between the first two queries MUST be at least one second.
+ * 2. The intervals between successive queries MUST increase by at least a factor of two.
+ *
+ * To adhere to these timing requirements for continuous queries,
+ * the maximum number of attempts should be set to 1.
+ */
+#define MDNS_TRANSACTION_CONTINUOUS_QUERY_MAX 1
```

Comment:
doesn't matter, but given this is returned as unsigned you might as well make this unsigned too, instead of relying the implicit casting C will do. i.e. change 1 to 1U